Urcheon/Texset.py

```diff
@@ -16,10 +16,12 @@
 from collections import OrderedDict
 import logging
 import os
-import pytoml
 import shutil
 import subprocess
 import tempfile
+import toml
+
+# FIXME: do we need OrderedDict toml constructor here?
```

Comment:
I'll let you figure that out :X

Reply:
I did some test, and default non-ordered constructor is not faster anyway.